unity-ggjj/Assets/Scripts/GameState/GameStarter.cs

```diff
@@ -0,0 +1,50 @@
+using SceneLoading;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace GameState
+{
+    [RequireComponent(typeof(NarrativeGameState))]
+    public class GameStarter : MonoBehaviour
+    {
+        [SerializeField] private GameStartSettings _gameStartSettings;
+        [Tooltip("Assign a narrative script text asset here to have it play on scene start"), SerializeField] private TextAsset _debugNarrativeScriptTextAsset;
+    
+        private NarrativeGameState _narrativeGameState;
+    
+        private void Awake()
+        {
+            _narrativeGameState = GetComponent<NarrativeGameState>();
+        }
+
+        private void Start()
+        {
+            if (_debugNarrativeScriptTextAsset != null)
```

Comment:
```suggestion
            if (_debugNarrativeScriptTextAsset)
```
I think this should work the same without `!= null`

Reply:
As per the discussion on Discord I'm going to keep it as it is for now